src/backend/core/warren/conf.py

```diff
@@ -18,11 +18,11 @@ class ESClientOptions(BaseModel):
 class Settings(BaseSettings):
     """Pydantic model for Warren's global environment & configuration settings."""
 
-    # Elasticsearch
-    ES_HOSTS: Union[List[str], str] = None
-    ES_INDEX: str = "statements"
-    ES_INDEX_TIMESTAMP_FIELD: str = "timestamp"
-    ES_CLIENT_OPTIONS: ESClientOptions = ESClientOptions()
+    # LRS backend
+    LRS_HOSTS: Union[List[AnyHttpUrl], AnyHttpUrl]
+    LRS_BASIC_USERNAME: str
+    LRS_BASIC_PASSWORD: str
```

Comment:
Maybe prefixing configuration variable related to authentication with `LRS_AUTH` would make sens. WDYT?

```suggestion
    LRS_AUTH_BASIC_USERNAME: str
    LRS_AUTH_BASIC_PASSWORD: str
```
By extension: `LRS_AUTH_OIDC_xxx`